website/docs/for-capacitor/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Overview
+sidebar_label: Overview
+---
+
+## Why use micro-frontends
+
+Traditional development of a monolithic frontend applications have required all developers to collaborate within a single codebase and also ship updates together using a release process. As an application grows over time and more teams are added this can slow the development of the overall system.
+
+One way of alleviating this is to divide the larger monolith into smaller applications. This way each team can develop in isolation and then ship updates on their own timeline. There are many existing tools for frontend developers that do just that. The most common is [Module Federation](https://webpack.js.org/concepts/module-federation/). This tool allows for teams to develop web applications independently and then bring the builds dependencies together at run-time. It works great for hosted web applications. For most teams a tool like this is probably not needed, but when you have multiple teams trying to collaborate and ship updates independently it can offer a lot of autonomy and value.
+
+## Purpose of Portals for Capacitor
+
+These tools work great for the web because web based micro-frontends developers can deploy real time. And each application lives on a different server or directory location so this separation is inherent to the web. But the same can not be said for mobile applications.
```

Comment:
The flow of this sentence is a little difficult. Maybe something along the lines of:

Micro-frontends work great for the web because developers can deploy in real time.  Each application lives on a different server or directory location so this separation is inherent on the web, but the same can not be said for mobile applications.